### ISSUE TITLE
variable_scope()got multiple values for keyword argument 'reuse'

### ISSUE THREAD

**Issue by tphyhFighting** [davidsandberg/facenet#68]
Hi,when I start classifier training
python facenet_train_classifier.py --logs_base_dir ~/logs/facenet/ --models_base_dir ~/models/facenet/ --data_dir ~/datasets/facescrub/facescrub_mtcnnalign_182_160:~/datasets/casia/casia_maxpy_mtcnnalign_182_160 --image_size 160 --model_def models.inception_resnet_v1 --lfw_dir ~/datasets/lfw/lfw_mtcnnalign_160 --weight_decay 2e-4 --optimizer RMSPROP --learning_rate -1 --max_nrof_epochs 80 --keep_probability 0.8 --random_crop --random_flip --learning_rate_schedule_file ../data/learning_rate_schedule_classifier_long.txt --center_loss_factor 2e-5
I get a error  Traceback (most recent call last):
  File "facenet_train_classifier.py", line 324, in <module>
    main(parse_arguments(sys.argv[1:]))
  File "facenet_train_classifier.py", line 101, in main
    phase_train=phase_train_placeholder, weight_decay=args.weight_decay)
  File "/root/facenet/src/models/inception_resnet_v1.py", line 145, in inference
    dropout_keep_prob=keep_probability,reuse=reuse)
  File "/root/facenet/src/models/inception_resnet_v1.py", line 167, in inception_resnet_v1
    with tf.variable_scope(scope, 'InceptionResnetV1', [inputs],reuse=reuse):
  File "/usr/lib/python2.7/contextlib.py", line 84, in helper
    return GeneratorContextManager(func(*args, **kwds))
TypeError: variable_scope() got multiple values for keyword argument 'reuse'

How could I do this?

**Comment by davidsandberg** [comment]
Which version of tensorflow are you running? If I remember correctly I ran into the same problem with TF 0.10 but with 0.11 it is fine.

**Comment by tphyhFighting** [comment]
My tensorflow's  version is 0.10,and I change the version ,now it is fine.Thank you

**Comment by Dagalaki** [comment]
Hi,
I am running into the same problem. I explain below why I cannot move to Tensorflow 0.11.

My Tensorflow version is 0.9 The reason I use this version, is because I had to install Tensorflow on Jetson TX1 . Currently Tensorflow does not support aarch64 , someone though managed to install Tensorflow 0.9 on TX1. So I had to follow his instructions doing some hacks to make tensorflow stable on Jetson TX1.

You think I can solver the error mentioned in this issue by other means and not by moving to 0.11?
Any suggestions would be welcome!

Thanks a lot
